Code/CMakeLists.txt

```diff
@@ -316,6 +316,31 @@ if(SV_USE_OPENCASCADE)
 	#set(SV_INSTALL_EXTERNALS ON)
 	find_package(FREETYPE)
 endif()
+
+#-----------------------------------------------------------------------------
+# GDCM
+if(SV_USE_GDCM)
+	simvascular_external(GDCM DOWNLOADABLE)
+	mark_as_superbuild(GDCM_DIR)
+	mark_as_superbuild(GDCM_SOURCE_DIR)
+	set(USE_GDCM ON)
+	set(GLOBAL_DEFINES "${GLOBAL_DEFINES} -DSV_USE_GDCM")
+	#set(SV_EXTERNAL_SHARED_LIBS ${SV_EXTERNAL_SHARED_LIBS} GDCM)
+	#set(SV_INSTALL_EXTERNALS ON)
+endif()
+
+#-----------------------------------------------------------------------------
+# FREETYPE
+if(SV_USE_FREETYPE)
+	simvascular_external(FREETYPE DOWNLOADABLE)

```

Comment:
Can you add a git repository for us under SV containing the source code for gdcm-2.6.1 and freetype-2.6.3?  The source code is in SV16 under "src/BuildHelpers/Originals"?  This source code doesn't need patches (at least for Windows, untested on linux).  Might as well add it and fix the CMake build files to point to the right git.